src/TaskList.js

```diff
@@ -1,14 +1,39 @@
 class TaskList {
-  static #taskList = [
-    { description: 'aaa', completed: true, index: 1 },
-    { description: 'bbb', completed: true, index: 2 },
-    { description: 'ccc', completed: false, index: 3 },
-    { description: 'ddd', completed: false, index: 4 },
-  ];
-
-  static getTaskList() {
-    return this.#taskList;
+  static #taskList = [];
+
+  static #lsTaskList = 'taskList';
+
+  static getTaskList = () => TaskList.#taskList;
+
+  static setStore() {
+    localStorage.setItem(TaskList.#lsTaskList, JSON.stringify(TaskList.#taskList));
+  }
+
+  static getStore() {
+    TaskList.#taskList = JSON.parse(localStorage.getItem(TaskList.#lsTaskList) || '[]');
   }
+
+  static add = (task) => {
+    const index = TaskList.#taskList.length + 1;
+    const newTask = { ...task, completed: false, index };
+    TaskList.#taskList.push(newTask);
+    TaskList.setStore();
+    return newTask;
+  };
+
+  static get = (index) => TaskList.#taskList[index - 1];
+
+  static modify = (task) => {
+    const { index } = task;
+    TaskList.#taskList[index - 1] = task;
+    TaskList.setStore();
+  };
+
+  static remove = (index) => {
+    const list = TaskList.#taskList.filter((task) => task.index !== index);
+    TaskList.#taskList = list.map((task, i) => ({ ...task, index: i + 1 }));
+    TaskList.setStore();
+  };
```

Comment:
Currently, the app is running fine, I can add a todo, and edit a todo but the delete functionality is not working.

- [ ] Kindly work on the delete method to be able to delete a todo from the list of todos and it should update the todos index in the localStorage when a todo is deleted by its index.

I'll suggest you check on the delete button and see if you added the click command on the button and also the delete method. 👍

Reply:
@AmaduKamara 
I want to express my frustration by your kindly request on the delete method, because what you suggest is not the way the original website behaves. The requirements document state that our application should "behave like the part of minimalist project" (screenshot included at the end).

I discover in the [original project](https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/) that **_the tasks are not deleted by clicking on the trash can icon, but when the description is empty_**, so I did it that way.

I submitted a [video](https://www.loom.com/share/75ce4fad64ec407f9c4a0f0cab674614) showing the behavior of the delete functionality both in my project and also in the original website.

The video also shows the indexes are being updated after a deletion, so I do not understand that part of your suggestion.

Of course, I can make those changes, but that would be departing from the requirements we were given.

![image](https://user-images.githubusercontent.com/100373071/165649701-d1f32f68-d602-478e-94d4-b77dac650acf.png)

Reply:
Hello @rbanos-mv!
Actually, this was a misunderstanding, you actually implemented the task according to the original design. 
Kindly resend your project for review and your project will be approved. You did a great job. 👍